core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -100,9 +101,19 @@ oppia.directive('topNavigationBar', [function() {
           $scope.$apply();
           // Close the sidebar, if necessary.
           SidebarStatusService.closeSidebar();
+          $scope.toggleText = '\uE5D2';
         });
 
-        $scope.toggleSidebar = SidebarStatusService.toggleSidebar;
+        $scope.toggleSidebar = (function() {
+          $scope.isSidebarShown = SidebarStatusService.isSidebarShown();
```

Comment:
@seanlip, i added ng-if tag. Even added css to close icon as it was shifting below when compared to hamburger icon.